modules/cli-installing-cli.adoc

```diff
@@ -63,9 +65,7 @@ command-line interface. You can install `oc` on Linux, Windows, or macOS.
 
 [IMPORTANT]
 ====
-If you installed an earlier version of `oc`, you cannot use it to complete all
-of the commands in {product-title} {product-version}. Download and
-install the new version of `oc`.
+If you installed an earlier version of `oc`, you cannot use it to complete all of the commands in {product-title} {product-version}. Download and install the new version of `oc`.
```

Comment:
I know this is intentionally vague, but is there a specific release that works? Can you say oc v1.0+, for example?

Reply:
TBH. I cannot answer. I do not own this module. I simply reuse it in the context of installation assemblies.